src/Stratis.SmartContracts.CLR/SCL/Operations.cs

```diff
@@ -1,9 +1,37 @@
 ﻿using System;
+using Stratis.SmartContracts;
 
 namespace Stratis.SCL.Base
 {
     public static class Operations
     {
-        public static void Noop() { }
+        public static Address DeserializeAddress(byte[] address)
```

Comment:
Why not copy the much more readable [existing implementation](https://github.com/stratisproject/StratisFullNode/blob/master/src/Stratis.SmartContracts.CLR/AddressExtensions.cs#L49)?

Reply:
Also this would go in the `Stratis.SCL.Serialization` namespace.

Reply:
Refactored.